packages/components-react/src/icon/Icon.tsx

```diff
@@ -0,0 +1,26 @@
+import IconSet, { type IconId } from '@lux-design-system/icons';
+// import * as IconsTabler from '@tabler/icons-react';
+import { Icon as UtrechtIcon, type IconProps as UtrechtIconProps } from '@utrecht/component-library-react';
+import { PropsWithChildren, ReactNode } from 'react';
+import FallbackIcon from './FallbackIcon';
+
+export type IconProps = UtrechtIconProps & {
+  iconId: IconId | string;
+  iconLibrary?: Partial<Record<string, ReactNode>>;
+};
+
+export const Icon = ({ iconId, iconLibrary, children }: PropsWithChildren<IconProps>): ReactNode => {
```

Comment:
```suggestion
export const LuxIcon = ({ iconId, library, children }: PropsWithChildren<IconProps>): ReactNode => {
```

Reply:
Gebruik de `Lux` prefix

Reply:
de `icon` prefix is voor props nit nodig, tenzij de prop dan een reeds bestaand HTML attribuut overschrijft